### DISCUSSION TITLE
Global and App CSS

### DISCUSSION THREAD

**Discussion by trashidev** [Budibase/budibase#4614] [Ideas]
I'd love to have more control over the CSS. A way to add CSS on global and app levels. Just this would allow so much more customization, especially fine-tuning mobile styles.

**Comment by mjashanks** [comment]
Definitely - something that is on the Radar. We're not sure how this will be implemented yet - whether it will be direct CSS, or user interface controlled. 

Global styling will be a paid / non-open-source feature.

**Reply by CovantaDataManagement** [reply]
Hey Mike - since we are a paid customer - any idea when we can have this?

**Reply by mjashanks** [reply]
Absolutely - we'll get together to understand what you need, then we can talk about when we can deliver.
